scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 654,
```

Comment:
According to running `npm run build` our smallest bundle is now 710 kB. Do we need to update the `MIN_SIZE` too?

Reply:
yes, we should. will do now.